src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -129,4 +154,12 @@ interface IEigenPodManager is IEigenPodManagerErrors, IEigenPodManagerEvents, IS
 
     /// @notice returns canonical, virtual beaconChainETH strategy
     function beaconChainETHStrategy() external view returns (IStrategy);
+
+    /**
+     * @notice Returns the amount of reported slashing a staker has experienced on the
```

Comment:
"Amount" is a bit unclear here as its not actual ETH amounts. Maybe historical sum of proportions slashed is more correct but also a mouthful.

Reply:
[678abaf](https://github.com/Layr-Labs/eigenlayer-contracts/pull/892/commits/678abaf2c00c20254daf9b4515dca86cae2495d3)